dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClasspathCheckOption.java

```diff
@@ -123,21 +129,30 @@ private static Options configureOptions() {
     return options;
   }
 
-  static ImmutableList<Path> generateInputClasspath(CommandLine commandLine)
-      throws RepositoryException, ParseException {
-    setRepositories(commandLine);
-
+  static ImmutableList<Artifact> generateArtifacts(CommandLine commandLine)
+      throws RepositoryException {
+    Preconditions.checkArgument(
```

Comment:
This is user error, right? I'm not sure `IllegalArgumentException` is the right one. Is there a `ParseException` subclass that's more appropriate? `MissingOptionException`?

(By the way, could you use an `OptionGroup` to make Apache report an error if more than one of `-b`, `-a`, and `-j` is used?)

Reply:
This error is for developer (me), when this method is called without having BOM or Maven artifacts.

OptionGroup worked good for this case. Thanks.

> The option 'b' was specified but an option from this group has already been selected: 'a'

Nice message.